src/docfx/lib/git/GitUtility.cs

```diff
@@ -222,7 +222,16 @@ public static void CheckMergeConflictMarker(string content, string file)
                 content.Contains("\n>>>>>>>") &&
                 content.Contains("\n======="))
             {
-                throw Errors.MergeConflict(file).ToException();
+                var start = content.IndexOf("<<<<<<<");
```

Comment:
can we do some optimization here? seems one content will be looped checked multiple times.